### DISCUSSION TITLE
hi i have a problem in card component

### DISCUSSION THREAD

**Discussion by koushiar** [chakra-ui/chakra-ui-docs#273] [Q&A] [answered]
i wanna create a component for card and for example i wanna pass onclick or background or ... in typescript
how can i do that

**Comment by noobinthisgame** [comment, marked as answer]
Hey @kourosh-md  :) 
you can pass the props of the component you are using to build the custom card component like this:

```typescript
export const Card : FunctionComponent<BoxProps> = ({children, ...props}: BoxProps) => {
   return <Box {...props}>{children}</Box>
}
```

This way you can use all BoxProps on your custom card component and they will be passed down.
